examples/tutorials/04_linear_models_example.ipynb

```diff
@@ -24,17 +24,21 @@
     "import warnings\n",
```

Comment:
<p>Do we really need these classes? We can do better with a simple layout <code>f</code>.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/analysiscenter/batchflow/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='23'/>

Reply:
Indeed we can define the models here through the config.
Classes tho might be a good example, that you can define the `body` as the small separate module, implemented directly in the notebook.